### DISCUSSION TITLE
Keep issues as issues until proven otherwise

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#14026] [Feature Requests]
The, "Well, it doesn't happen for me so it isn't a bug, but instead, let's move it to discussions so it doesn't look like we have a lot of issues and also where they'll move down and be forgotten" isn't really conducive to good development practises. A discussion can happen in an issue.

I think this workflow is a "bug" and needs ironing out!

**Comment by benhaynes** [comment]
I understand the frustration of something reported be deescalated, but let me provide some insight into our process.

* The "_It doesn't happen to me..._" is not the case, it's more "_This is not reproducible given the current information_".
* Other times, these "issues" are really config problems, and these issues are usually closed with hours/days by the OP due to some "holding it wrong" config issue.
* Issues being converted to Discussions does not mean they are not being considered or will be forgotten. It simply is another layer of triage and prioritization. Usually, it means they need more... _discussion_ and community feedback before a solution can be made. Yes, discussion can happen on an issue, but the Discussions have _threaded_ comments, which are better for adding/exploring multiple possible solutions.
    * Also, this does indeed keep the issue count down, but that is not for _optics_, that is to help us focus on the main issues in Core. This may seem trivial, but we close at least 50% of all reported issues as "Duplicates" of existing tickets. That is a LOT of work, which can be reduced if there are fewer pages/issues for users to search through when reporting something new.
* Lastly, certain reports are for very specific environments or usage within an extreme edge case. While this may actually be a bug, our goal is to keep the codebase as clean and maintainable as possible. To that end, we want to make sure it passes our 80/20 rule. 
    * Directus is a _very_ broad platform, since it can do almost anything, almost anything could be considered a "bug" if it doesn't conform to the user's expectations. To avoid bloat, we want to ensure that the use-case is _relatively_ standard usage. Also, the definition of this "standard usage" will expand over time.

**Reply by benhaynes** [reply]
Also, to add one more... as maintainers of this software, we're doing our best to stay organized so our small team can stay on top of some pretty crazy growth (and prepare for even more growth). This process is the direction we've chosen... however, it goes without saying that any Discussion that an individual or group of users see as a _bug_ can still be resolved via a contribution! 🙏

**Comment by rijkvanzanten** [comment]
We keep issues as the actionable items (eg todos) we can assign between contributors on our projects boards and other internal project management tooling. Having items floating around in there that we can't really operate against (eg things that we haven't been able to reproduce at all) adds a ton of managerial overhead. To add onto that, more often than not those particular issues turn out to be a hosting/configuration/usage problems. 

> [...] so it doesn't look like we have a lot of issues and also where they'll move down and be forgotten

I don't appreciate your assumptions here.. The way we triage, assign, and resolve is based on what works best for us as a small OSS team, and has nothing to do with appearances. Besides, as you can tell by me responding here immediately, we don't ignore _Discussions_ either. As a matter of fact, the whole core team watches the whole repo so gets notified of everything 🤷🏻 

> I think this workflow is a "bug" and needs ironing out!

IMO this workflow has been working great in keeping any reasonable sense of organization and ability to actually maintain this platform at scale. Keeping everything under issues (including questions, feature requests, and not-reproducible issues; like we used to do in prior versions) means you end up with hundreds/thousands open issues, which actually ends up burying the actionable items 🙂
